CMake/AbseilFindDependencies.cmake

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include(ExternalProject)
+
+
+#
+## pthread
+#
+find_package(Threads REQUIRED)
+
+#
+## CCTZ
+#
+if(ABSL_NO_BUNDLE)
+    find_package(CCTZ REQUIRED)
+else(ABSL_NO_BUNDLE)
+
+    # if not found, try to bundle it
+    message(STATUS "Use bundle CCTZ package")
+
+    ## CCTZ has no cmake support YET
```

Comment:
FYI, I have submitted a PR for CMake support in cctz:
- https://github.com/google/cctz/pull/54